.travis.yml

```diff
@@ -15,16 +15,18 @@ addons:
     - gcc
     - git
     - libgmp-dev
-    - libllvm5.0
+    # - libllvm5.0
```

Comment:
Any reason why these and below are commented out?

Reply:
> Any reason why these and below are commented out?

Because we don't use llvm code generator.

Reply:
Maybe I should have commented on the lines below, but `llvm-5.0` and `llvm-5.0-dev` are both removed, and afaik at least one of those provides a linker in the environment. You mentioned in another comment that Travis is broken, I wouldn't be entirely surprised if this was why.

Reply:
I doubt it. llvm is only used for llvm code generation. The linker is provided by binutils.